src/app/helper/inbox-linshare-helper.service.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinshareHelper', inboxLinshareHelper);
```

Comment:
This will cause an issue in Webpack production mode. It should be `inboxLinshareHelperFactory`